packages/allure-js-commons/src/ExecutableItemWrapper.ts

```diff
@@ -16,7 +16,7 @@ import {
 export class ExecutableItemWrapper {
   constructor(private readonly info: FixtureResult | TestResult) {}
 
-  protected get wrappedItem(): FixtureResult | TestResult {
```

Comment:
why change?

Reply:
it was necessary when i had computed `historyId`. cleaned up